src/main/java/com/debatetimer/controller/member/dto/TableResponses.java

```diff
@@ -0,0 +1,23 @@
+package com.debatetimer.controller.member.dto;
+
+import com.debatetimer.domain.parliamentary_debate.ParliamentaryTable;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+
+public record TableResponses(
```

Comment:
ArrayResponse의 경우 제네릭으로 선언하여 사용하는 건 어떨까요?

```suggestion
public record ArrayResponse<T>(
        List<T> responses
    )
```

Reply:
api에 responses 대신에 tables로 되어 있어서 TableResponses로 만들었습니다.

> ArrayResponse의 경우 제네릭으로 선언하여 사용하는 건 어떨까요?

모든 배열 응답 값이 responses로 의무적으로 통일하는 것도 썩... 좋진 않은 것 같습니다. 콜리의 생각은 어떤가요?